File: ecommerce/src/main/java/com/acme/ecommerce/dtos/SaleDTO.java
package com.acme.ecommerce.dtos;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

/**
 * @author s2it_csilva
 * @version : $<br/>
 * : $
 * @since 11/15/19 10:17 AM
 */
@Data
@Builder
@AllArgsConstructor
public class SaleDTO {
    private Long idProduct;
    private Integer amount;
}

File: stock/src/main/kotlin/com/ecma/stock/dtos/SaleDTO.kt
package com.ecma.stock.dtos

import com.ecma.stock.enumeration.Transaction
import java.time.LocalDateTime

/**
 * @author s2it_csilva
 * @since 11/15/19 10:12 AM
 * @version : $<br/>
 *          : $
 *
 */
data class SaleDTO (
        var idProduct: Long,
        var amount: Int,
        var date: LocalDateTime?,
        var transaction: Transaction?
){
    constructor() : this(0, 0, null, null) {}
}